src/acktest/bootstrapping/vpc.py

```diff
@@ -0,0 +1,185 @@
+from typing import List, Union
+import boto3
+import logging
+
+from boto3.session import Session
+from dataclasses import dataclass, field
+
+from . import BootstrappableResource
+from .. import resources
+from ..aws.identity import get_region
+
+# Subnets inside the default VPC CIDR block will be of form 10.0.*.0/24
+VPC_CIDR_BLOCK = "10.0.0.0/16"
+
+
+@dataclass
+class VPC(BootstrappableResource):
```

Comment:
This sets up a lot more than just a VPC. It sets up subnets, internet gateways, route tables, etc. This isn't always required for some simple fixtures. Recommend renaming this resource `VPCPublicInternet` or similar and having a simpler `VPCPrivate` that doesn't set up the public stuff.